db/seeds/source_types.yml

```diff
@@ -123,6 +123,34 @@ ansible-tower:
         :condition:
           :when: endpoint.verify_ssl
           :is: true
+google:
+  :product_name: Google Cloud Engine
+  :vendor: Google
+  :icon_url: "/apps/chrome/assets/images/partners-icons/google-cloud-engine.svg" # TODO
```

Comment:
I have already uploaded the logo: '/apps/frontend-assets/partners-icons/google-cloud.svg'